povm_toolbox/library/povm_implementation.py

```diff
@@ -171,6 +173,38 @@ def compose_circuits(self, circuit: QuantumCircuit) -> QuantumCircuit:
                 f" {self.num_qubits} qubits."
             )
 
+        # Check if the measurement circuit requires some ancilla qubits
+        if self.measurement_circuit.num_qubits > self.num_qubits:
+            index_layout = copy(index_layout)
+            # Get idle qubits in supplied circuit (to be used as ancilla for measurement circuit)
+            idle_qubits = list(circuit_to_dag(dest_circuit).idle_wires())
+            # Get the indices of the idle qubits
+            idle_index = [dest_circuit.qubits.index(qubit) for qubit in idle_qubits]
+            # Remove the idle qubits that will be measured (as specified by index_layout)
+            idle_index = [idx for idx in idle_index if idx not in index_layout]
+            # TODO: why would someone want to measure an idle qubit ? Should we raise a warning ?
```

Comment:
Let's raise an error when this happens. I'd rather have this not be supported at all then to overwrite a case a user might have triggered specifically.
If a user would like this to work, they can open an issue to request this to be working.

Reply:
These lines are visited only if the measurement circuit requires some ancilla qubits. Should we move this check outside this `if` statement (i.e., check/raise all the time ?)